internal/security/vaultclient/interface.go

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright (c) 2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+//
+// SPDX-License-Identifier: Apache-2.0'
+//
+
+package vaultclient
```

Comment:
personally I would like it if we reduce the amount of package naming specific to the implementation, i.e. we use registry package instead of vault, so maybe this could be 

```suggestion
package secretstoreclient
```

Reply:
Changed as suggested.